docs/cli/debug.md

```diff
@@ -116,16 +116,17 @@ Options:
                             ANTA_ENABLE]
   -P, --prompt              Prompt for passwords if they are not provided.
                             [env var: ANTA_PROMPT]
-  --timeout INTEGER         Global connection timeout  [env var: ANTA_TIMEOUT;
-                            default: 30]
-  --insecure                Disable SSH Host Key validation  [env var:
+  --timeout FLOAT           Global API timeout. This value will be used for
+                            all devices.  [env var: ANTA_TIMEOUT; default:
+                            30.0]
+  --insecure                Disable SSH Host Key validation.  [env var:
                             ANTA_INSECURE]
-  --disable-cache           Disable cache globally  [env var:
+  --disable-cache           Disable cache globally.  [env var:
                             ANTA_DISABLE_CACHE]
-  -i, --inventory FILE      Path to the inventory YAML file  [env var:
+  -i, --inventory FILE      Path to the inventory YAML file.  [env var:
                             ANTA_INVENTORY; required]
-  -t, --tags TEXT           List of tags using comma as separator:
-                            tag1,tag2,tag3  [env var: ANTA_TAGS]
+  --tags TEXT               List of tags using comma as separator:
```

Comment:
ouch breaking change :p

Reply:
need to make something in therelease notes or whatever

Reply:
Yup :)

Reply:
lets' start by adding a `!` here